ui/DesignSystem/Component/SourceBrowser/History.svelte

```diff
@@ -45,21 +46,23 @@
   }
 </style>
 
-{#each history.history as group (group.time)}
-  <div class="commit-group">
-    <header>
-      <p>{group.time}</p>
-    </header>
-    <ul>
-      {#each group.commits as commit (commit.sha1)}
-        <li class="commit" on:click={() => onSelect(commit)}>
-          <CommitTeaser
-            {commit}
-            style="background: none; --commit-message-color:
+<div class="history" data-cy="history">
```

Comment:
:information_source: the only change here is wrapping the list of commit groups in a `div.history` element.

Reply:
My bad, the previous rebase & merge-resolution didn't go well. Just rebased again and now it's all in good shape.

So, answering your question, see https://github.com/radicle-dev/radicle-upstream/pull/1686/files#diff-c1135bec4b86c0f227a805cabfd0d9933c669329ede40a2ef36a2c031c066264R423. The list of commit groups wasn't wrapped in an element so testing the number of commits would be unreliable as if we had more group-commits displayed anywhere else in the page things wouldn't add up.

Reply:
:heavy_check_mark: